pkg/controllers/dynakube/extension/eec/reconciler_test.go

```diff
@@ -82,6 +84,24 @@ func getStatefulset(t *testing.T, dk *dynakube.DynaKube) *appsv1.StatefulSet {
 	return statefulSet
 }
 
+func mockTLSSecret(t *testing.T, client client.Client, dk *dynakube.DynaKube) client.Client {
```

Comment:
I can't find the test. When we update the secret, should we run the reconciler and make sure the annotation is updated 🤔 ?
IMO, it's the main use case of the story.

Reply:
[Fixed!](https://github.com/Dynatrace/dynatrace-operator/pull/3843/commits/4cb0c88c4d0d164a77c82ce5ea5ec561a7b1ae3f) Added very nice tests :)